java/src/shared/model/player/Player.java

```diff
@@ -702,6 +702,13 @@ public void setDiscarded(boolean discarded) {
         this.discarded = discarded;
     }
 
+    public void setVictoryPoints(int points){
+        assert points >=0;
+        assert  points <=10;
```

Comment:
Shouldn't it be possible for players to get over 10 points? Over 10 points just indicates victory. For example if someone has 9 points then plays some dev card to get more than 10 points? cc. @yeyyosing